packages/laconia-event/src/DynamoDbStreamRecord.js

```diff
@@ -1,18 +1,36 @@
-const AWS = require("aws-sdk");
+require("aws-sdk/clients/dynamodb");
```

Comment:
Why this is better?
It's more about what I can learn from your PR because a never see this before (sorry) :-)

I think this could be better for tree shaking but is ugly (AWS-SDK that is, not your suggestion).

Reply:
Agreed it's unsightly!!! It's based on this article https://theburningmonk.com/2019/03/just-how-expensive-is-the-full-aws-sdk/